### PR TITLE
AC Test Fix - LRAC-10829 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10829
[Testray](https://testray.liferay.com/home/-/testray/case_results/index?p_p_state=normal&orderByCol=status_sortable&orderByType=asc&testrayBuildId=1652580111)

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Review by manuferreira — APPROVED** [review]
The code looks good to me Vini, very good looking `viewTimelineItem` macro :grin:

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACIndividualsDashboard.macro [hunk 1]
```diff
@@ -305,18 +305,42 @@ definition {
...
+		var i = "0";
+
+		while ("${i}" != "${size}") {
+			var activityName = ListUtil.get("${list1}", "${i}");
+			var position = ListUtil.get("${list2}", "${i}");
+
+			AssertElementPresent(
+				index = "${position}",
+				key_commonInformation = "${activityName}",
+				locator1 = "ACEventAnalysis#VIEW_ANALYSIS_INFORMATION_SPECIFC_ORDER");
```
Comment (christianstokes) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACIndividualsDashboard.macro:320]: Typo:

_SPECIFC_
should be
_SPECIFIC_

**Review by christianstokes — CHANGES_REQUESTED** [review]
@ViniLopes20 the macro looks great! There is one typo in the path locator: VIEW_ANALYSIS_INFORMATION_**SPECIFC**_ORDER.

According to my search, it's only used by this macro. Can you fix it? Thanks!

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACIndividualsDashboard.macro (hunk 1 above)
Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACIndividualsDashboard.macro:320]: Good catch, Chris. Thanks 😄

**Comment by ViniLopes20** [comment]
ci:forward

**Comment by ViniLopes20** [comment]
Hi team, Thanks for reviewing!

I'm forwarding it!

**Pull request closed without merging**